src/select/select.component.ts

```diff
@@ -4,7 +4,9 @@ import { Component, Input } from "@angular/core";
 	selector: "ibm-select",
 	template: `
 		<div class="bx--form-item">
-			<div class="bx--select">
+			<div
+				[ngClass]="{ 'bx--select--inline': inlineSelect }"
```

Comment:
All other places in the codebase do
```suggestion
				[ngClass]="{'bx--select--inline': inlineSelect}"
```